README.md

```diff
@@ -155,6 +161,17 @@ try
 catch e
     println(e)  # ErrorException("assertion failed: istrait(Tr4{Int,Float64})")
 end
+
+# for parametric trait,
+@traitimpl SemiFunctor{Nullable{T}} begin
+    fmap{T}( f::Function, x::Nullable{T}) = Nullable(f(x.value))
+end
```

Comment:
This implementation of `fmap` will make a non-null value out of a null value:
```
julia> Nullable(sin(Nullable{Int}().value))
Nullable(0.8414709848078965)
```
Is that intended?